upgrade_through_versions_test.py

```diff
@@ -877,33 +715,18 @@ def __init__(self, *args, **kwargs):
         kwargs['cluster_options'] = {'partitioner': 'org.apache.cassandra.dht.RandomPartitioner'}
         Tester.__init__(self, *args, **kwargs)
 
-    @property
-    def test_versions(self):
-        return [make_branch_str(v) for v in UPGRADE_PATH]
 
-
-class PointToPointUpgradeBase(TestUpgradeThroughVersions):
+class BootstrapMixin(object):
     """
-    Base class for testing a single upgrade (ver1->ver2).
+    Can be mixed into UpgradeTester or a subclass thereof to add bootstrap tests.
 
-    We are dynamically creating subclasses of this for testing point upgrades, so this is a convenient
-    place to add functionality/tests for those subclasses to run.
+    Using this class is not currently feasible on lengthy upgrade paths, as each
+    version bump adds a node and this will eventually exhaust resources.
 
     __test__ is False for this class. Subclasses need to revert to True to run tests!
     """
     __test__ = False
```

Comment:
If this is going to be a mixin, we probably (probably?) don't want to define this kind of test object behavior here. I would expect it to just attach some functions and that's it.